includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -621,11 +621,10 @@ public function header_logo() {
 	public function get_settings_text( $settings_key, $default = false, $autop = true ) {
 		if( array_key_exists( 'default', $this->settings[$settings_key] ) ) {
 			$text = wptexturize( trim( $this->settings[$settings_key]['default'] ) );
-		} else {
+		} elseif( ! empty( $this->settings[$settings_key] ) && is_array( $this->settings[$settings_key] ) ) {
```

Comment:
if `'default'` is `empty`, it will get to this step, and still meet both requirements. But to keep the current behavior it should then use `$default`.